doc/manual/manual.tex

```diff
@@ -1339,21 +1354,234 @@ \subsection{Numerical methods}
+subsections make use of the fact that for the flows for which these approximations are valid, the
+perturbations $p'$, $T'$ are much smaller than typical values of the reference
+quantities $\bar p$, $\bar T$. 
+The terms influenced by these approximations are $\nabla \cdot (\rho u) =0$ in the 
+continuity equation, and all occurrences of $\rho(p,T)$ in the temperature equation, 
+and we will discuss them separately below. The equations for these approximations are
+almost always given in terms of non-dimensionalized quantities. We will for
+now stick with the dimensional form because it expresses in a clearer way the
+approximations that are made. The non-dimensionalization can then be done on
+each of the forms below separately.
+
+\subsubsection{The anelastic liquid approximation (ALA)}
+\label{sec:ala}
+
+The \textit{Anelastic Liquid Approximation (ALA)} is based on two assumptions.
+First, that the density variations $\rho(p,T)-\bar\rho$ are small and in
+particular can be accurately described by a Taylor expansion:
+\begin{align*}
+  \rho(p,T) \approx 
+  \bar\rho 
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial p} p'
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T'.
+\end{align*}
+Here, $\frac{\partial \rho(\bar p,\bar T)}{\partial T}$ is related to the
+thermal expansion coefficient, and $\frac{\partial \rho(\bar p,\bar T)}{\partial p}$ to the
+compressibility.
+
+The second assumption is that the variation of the density
+from the reference density can be neglected in the mass balance and
+temperature equations.
+This yields the following system of equations for the velocity and pressure
+equations:
+\begin{align}
+  \label{eq:stokes-ALA-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  \left(\frac{\partial \rho(\bar p,\bar T)}{\partial p} p'
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T' \right) \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-ALA-2}
+  \nabla \cdot (\bar\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+
+For the temperature equation, using the definition of the hydrostatic pressure gradient \eqref{eq:hydrostatic-pressure}, we arrive at the following:
+
+\begin{multline}
+  \label{eq:temperature-decomposed}
+  \bar\rho C_p \left(\frac{\partial T}{\partial t} + \mathbf u\cdot\nabla
+  T\right) - \nabla\cdot k\nabla T
+  \\
+  =
+  \bar\rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha \bar\rho T (\mathbf u \cdot \mathbf g)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+
+\note{Our energy equation is formulated in terms of $T$, while in the literature, people 
+have chosen to formulate the equation in terms of $T'$, which yields additional terms 
```

Comment:
"people" is a bit informal. How about "...while in the literature, the equation has sometimes been formulated in terms of $T'$, then yielding additional terms..."